rust-runtime/aws-smithy-types/src/date_time/mod.rs

```diff
@@ -596,4 +601,50 @@ mod test {
         assert!(fourth > third);
         assert!(fourth == fourth);
     }
+
+    #[cfg(all(
+        test,
+        aws_sdk_unstable,
+        feature = "serde-deserialize",
+        feature = "serde-serialize"
+    ))]
+    #[test]
+    fn human_readable_datetime() {
```

Comment:
Consider moving these to the corresponding ser/de modules so that the cfg flag isn't needed on them.

Reply:
ok

Reply:
Let me fix it up.

Reply:
fixed.